README.md

```diff
@@ -20,11 +20,11 @@ please feel free to read through this page if you're interested in how Bash-it l
 
 ## Install
 
-1. Check out a clone of this repo to a location of your choice, such as:
+1. Check out a clone of this repo to a location of your choice, such as
    `git clone --depth=1 https://github.com/Bash-it/bash-it.git ~/.bash_it`
 2. Run `~/.bash_it/install.sh` (it automatically backs up your `~/.bash_profile` or `~/.bashrc`, depending on your OS)
 3. Edit your modified config (`~/.bash_profile` or `~/.bashrc`) file in order to customize Bash-it.
-4. Check out available aliases, completions and plugins and enable the ones you want to use (see the next section for more details).
+4. Check out available aliases, completions, and plugins and enable the ones you want to use (see the next section for more details).
```

Comment:
Yay oxford comma! Can't believe I missed it. :-)

Reply:
Happens :-)